pkg/config/nodetreemodel/config_test.go

```diff
@@ -343,6 +343,22 @@ func TestIsSet(t *testing.T) {
 	assert.False(t, cfg.IsKnown("unknown"))
 }
 
+func TestIsConfigured(t *testing.T) {
+	cfg := NewConfig("test", "TEST", nil)
```

Comment:
Since a key difference between IsSet and IsConfigured is how env vars are handled, we should have something in this test related to env vars.